ingest/monitoring/metrics_service.py

```diff
@@ -0,0 +1,33 @@
+# Service to record events to Mixpanel via Bard
+#
+# Mixpanel is a business analytics service for tracking user interactions.
+# Bard is a DSP service that mediates writes to Mixpanel.
+#
+
+import json
+import requests
+
+
+class MetricsService:
+    BARD_HOST_URL = "https://terra-bard-dev.appspot.com/api/event"
```

Comment:
We're going to need a way for this to work in all environments, so hard-coding the URL like this won't work.

Reply:
I suggest passing `BARD_HOST_URL` as an input parameter to `ingest_pipeline.py`.  This would let us keep one central list of environment-to-Bard-host mappings, [as defined in `single_cell_portal_core`](https://github.com/broadinstitute/single_cell_portal_core/search?q=bard_host_url&type=code).

Not using this suggested approach caused a bug in another SCP Bard client; see SCP-2866.